Source/Renderer/Texture.js

```diff
@@ -603,11 +616,12 @@ Object.defineProperties(Texture.prototype, {
 /**
  * Copy new image data into this texture, from a source {@link ImageData}, {@link HTMLImageElement}, {@link HTMLCanvasElement}, or {@link HTMLVideoElement}.
  * or an object with width, height, and arrayBufferView properties.
- *
- * @param {Object} source The source {@link ImageData}, {@link HTMLImageElement}, {@link HTMLCanvasElement}, or {@link HTMLVideoElement},
+ * @param {Object} options Object with the following properties:
```

Comment:
For the sake of API symmetry we should also make the same changes to `CubeMapFace.prototype.copyFrom`.